app/presenters/menu/yaml_loader.rb

```diff
@@ -26,23 +26,38 @@ def load_custom_item(file_name)
       end
     end
 
+    # In case `rbac` is a Hash, convert keys to symbols.
+    #   Example: { :feature => 'vm_explorer', :any => true }
+    #
+    # Else assume string and return:
+    #   { :feature => rbac }
```

Comment:
The `rbac` in the comment here is bare...typo, but not sure if it should be a symbol or a string.